lib/db/redis.js

```diff
@@ -0,0 +1,75 @@
+var redis = require('redis')
+var jws = require('jws')
+var P = require('../promise')
+
+function match(filter, str) {
+  try { // too lazy
+    var payload = JSON.parse(jws.decode(str).payload)
```

Comment:
A plain `decode` isn't that expensive, a base64 decode and json parse. Only `verify` does the crypto bits. But we still ought to test it with some benchmarks. Good call.